### PR TITLE
Cloning GEOS_mksi instead for GSI channel records

### LINKED ISSUE #289: Split GSI's mksi into its own repo
To facilitate the transition to JEDI, it would be good to have mksi sit its own repo - but be placed in the same GSI_GridComp location as it stands when GEOSadas is built. 

Added Akira here so she can update swell once the change has been made.

### PR DESCRIPTION
**Pull request by asewnath**
## Description

Thanks to @mathomp4, we have a lightweight repository for accessing GSI channel records. 

Changes include:
- Renaming task references to CloneGeosMksi and task script is called clone_geos_mksi.py
- Modifying clone_geos_mksi.py to clone GEOS_mksi instead and updating all paths as necessary
- Adding ability to use a tag for GEOS_mksi (question shows up in experiment.yaml)

Closes #289

### CODE REVIEW

**Comment by Dooruk** [comment]
Thanks @asewnath for making this change swiftly and tnaks @mathomp4 for the GEOS-ESM repo.s==s.

Just a question for @rtodling. Does the following setup work as default, which uses the `geosadas` suite? That would be only way for me to test this PR or CI-workflow to test it.
https://github.com/GEOS-ESM/swell/blob/develop/src/swell/test/suite_tests/geosadas-tier1.yaml

File: src/swell/configuration/jedi/interfaces/geos_atmosphere/task_questions.yaml [hunk 1]
```diff
@@ -187,6 +187,9 @@ observations:
 observing_system_records_gsi_path:
   default_value: None
 
+observing_system_records_gsi_path_tag:
```
Comment (rtodling) [inline src/swell/configuration/jedi/interfaces/geos_atmosphere/task_questions.yaml]: I'd suggest renaming all these things that say gsi ... this is not gsi specifically. these could be renamed: 

...records_mksi_path ... etc

File: src/swell/tasks/clone_geos_mksi.py [hunk 2]
```diff
@@ -31,18 +31,23 @@ def execute(self):
 
         # Parse config
         # ------------
-        path_to_geosana_gridcomp = self.config.observing_system_records_gsi_path()
+        path_to_geos_mksi = self.config.observing_system_records_gsi_path()
```
Comment (rtodling) [inline src/swell/tasks/clone_geos_mksi.py]: here again ... gsi -> mksi

Comment (rtodling) [reply src/swell/tasks/clone_geos_mksi.py]: and right below

File: src/swell/tasks/generate_observing_system_records.py [hunk 3]
```diff
@@ -38,11 +38,10 @@ def execute(self):
             cycle_dir = self.cycle_dir()
             observing_system_records_path = os.path.join(cycle_dir, 'observing_system_records')
 
-        path_to_geosana_gridcomp = self.config.observing_system_records_gsi_path()
-        if path_to_geosana_gridcomp == 'None':
-            path_to_geosana_gridcomp = os.path.join(self.experiment_path(), 'GEOSana_GridComp')
-        path_to_gsi_records = os.path.join(path_to_geosana_gridcomp, 'GEOSaana_GridComp',
-                                           'GSI_GridComp', 'mksi', 'sidb')
+        path_to_geos_mksi = self.config.observing_system_records_gsi_path()
```
Comment (rtodling) [inline src/swell/tasks/generate_observing_system_records.py]: same

File: src/swell/tasks/task_questions.yaml [hunk 4]
```diff
@@ -607,10 +607,20 @@ observing_system_records_gsi_path:
   - geos_atmosphere
   prompt: What is the path to the GSI formatted observing system records?
   tasks:
-  - CloneGeosAna
+  - CloneGeosMksi
   - GenerateObservingSystemRecords
   type: string
 
+observing_system_records_gsi_path_tag:
```
Comment (rtodling) [inline src/swell/tasks/task_questions.yaml]: same

**Review by rtodling — COMMENTED** [review]
Aside from the minor comments I make below - this is looking good!

File: src/swell/configuration/jedi/interfaces/geos_atmosphere/task_questions.yaml (hunk 1 above)
Comment (asewnath) [reply src/swell/configuration/jedi/interfaces/geos_atmosphere/task_questions.yaml]: Will do

**Comment by asewnath** [comment]
> Thanks @asewnath for making this change swiftly and tnaks @mathomp4 for the GEOS-ESM repo.s==s.
> 
> Just a question for @rtodling. Does the following setup work as default, which uses the `geosadas` suite? That would be only way for me to test this PR or CI-workflow to test it. https://github.com/GEOS-ESM/swell/blob/develop/src/swell/test/suite_tests/geosadas-tier1.yaml

If it helps, I tested the changes locally using the ufo_testing suite

**Review by Dooruk — APPROVED** [review]
Tier1 tests work, so if @rtodling approves it's good to merge! Thank you @asewnath for the quick turnaround.

**Comment by rtodling** [comment]
Thank you!

**Pull request merged**